Description: Build with the CMake of Debian stretch (3.7)
 add_compile_definitions() came with CMake 3.12; add_definitions() does
 the same here.  Nothing else of the build needs more than 3.7, so the
 viewer package builds from stretch to trixie with the distribution's
 CMake.
Author: Peter Schlaile <peter@schlaile.de>
Forwarded: not-needed

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,7 +2,7 @@
 # Setup
 #
 
-cmake_minimum_required(VERSION 3.10.0)
+cmake_minimum_required(VERSION 3.7.0)
 
 # Internal cmake modules
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/Modules)
@@ -83,7 +83,7 @@
   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /W3")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /W3")
 else()
-  add_compile_definitions(_FORTIFY_SOURCE=2)
+  add_definitions(-D_FORTIFY_SOURCE=2)
   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -Wformat=2 -Wvla")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wformat=2 -Wvla")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wzero-as-null-pointer-constant")
